bookmarks.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App Bookmarks</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <h1>Bookmarks</h1>
```

Comment:
cool jelena...easy and simple 👍 i would recommend to wrap your `h1` in a `header`  tag

Reply:
Thank you 😊
I first had it, but Jerry commented in Andreas pull request that it is not necessary when just having one thing in it - 
Thats why I deleted it.🤷🏼‍♀️